ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -848,6 +848,36 @@ tests:
       enable:
       - observers-resource-watch
     workflow: network-flow-matrix-single-node-bm
+- as: aws-ovn-network-flow-matrix
+  cron: 0 23 * * 1
+  steps:
+    cluster_profile: aws
+    env:
+      USER_TAGS: |
```

Comment:
I'm not sure these tags are needed since we are cloning the repo in the test workflow